dataapiv1/spec.yaml

```diff
@@ -508,10 +590,42 @@ paths:
             X-CB-Flags:
               $ref: "#/components/headers/DocumentFlags"
           content:
+            'application/json':
```

Comment:
Same as above related to https://github.com/couchbase/stellar-gateway/blob/master/gateway/dapiimpl/server_v1/helpers.go#L74-L85